### ISSUE TITLE
cannot make it to work after update to 0.6.3

### ISSUE THREAD

**Issue by rubenixnagios** [joaovitoriasilva/endurain#52]
did not make any change on docker compose file...
endurain backend log:
```
MySQLdb.OperationalError: (2005, "Unknown server host 'mariadb' (-2)")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):

  File "/usr/local/lib/python3.12/site-packages/starlette/routing.py", line 693, in lifespan

    async with self.lifespan_context(app) as maybe_state:

               ^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (2005, "Unknown server host 'mariadb' (-2)")

(Background on this error at: https://sqlalche.me/e/20/e3q8)

ERROR:    Application startup failed. Exiting.
```
i can reach the frontpage:

![imatge](https://github.com/user-attachments/assets/a298da52-2af8-4be1-ab35-826d760929f8)

but cannot make login, not get into anything...
ive noticed permissions changed on filesystem for mariadb folder:
999 systemd-journal 4,0K 23 de nov.  11:51 mariadb/

hope to get a solution for this as I cannot get back to previous version....

**Comment by rubenixnagios** [comment]
ok that was my guess.
what permissions on filesystem needs mariadb folder in order to get containers to work again?

tx so much

**Comment by rubenixnagios** [comment]
ok fair enought, do i need to update any line on the docker compose yml
file?

just pulled the image and stuck in here...
i wanted to see the whole translation work i did...hehehe

**Comment by rubenixnagios** [comment]
ok just removed tc.log on database folder (backed it up) and seems to connect to DB but now I get following error at logon:

`It was not possible to connect to the server. Please try again later - SecurityError: The operation is insecure.`

no errors on backend nor frontend containers...
checked from phpmyadmin; data is there

anything I should check to get more info?

**Comment by joaovitoriasilva** [comment]
Can you share the docker compose file hiding sensitive info?

Are both connections browser to frontend and frontend to backend using https or only one is using SSL?

**Comment by joaovitoriasilva** [comment]
Hi again. I just also notified this error: "Please try again later - SecurityError: Failed to construct 'WebSocket': An insecure WebSocket connection may not be initiated from a page loaded over HTTPS."

I already know why it exists and it is already partially fixed in version "dev-23112024".
If you use that dev version that error should no longer appear. However this error should not break the app, only the WebSocket part that is being used for Garmin Connect login feature. 

In summary something is fixed but I don't expect to fix your specific issue. Once again I can't reproduce what your are reporting. 

Nevertheless if possible, please give me feedback on this.

**Comment by rubenixnagios** [comment]
thanks for ur answers, been investigating...

just found that its some kind of bad proxy configuration going on. error just happens once service gets exposed to internet. 

so....will have to investigate further on this.

main facts:
- all other services exposed work
- endurain worked since updated to 0.6.3
- if I revert back both containers to 0.6.2 can reach endurain from internet...

ok il keep you updated on this, again, many thanks

will have to stay on 0.6.2 without my translation hehe

gimme a shout if u think of somethink i can try...

**Comment by joaovitoriasilva** [comment]
Okay I was finally able to reproduce your error. Using Safari I did not see the thing, changed to Firefox, voila!

I just deployed dev-24112024. If you had already tried with this dev version, please re download the images.
Can you give me feedback if it works?

Thank you.

**Comment by rubenixnagios** [comment]
great job!!
thank got u find it, was getting mad at this

indeed dev_24112024 (not middle dash) working as expected. im on 0.6.4 now and everything works!

tx again & keep this work on!!

**Comment by joaovitoriasilva** [comment]
Awesome. And sorry about this.
v0.6.4 released with the fix.
